src/services/gitpod.ts

```diff
@@ -1,36 +1,31 @@
 import { Buffer } from 'buffer';
 import type { RefreshTokenData } from 'src/types';
 
-import { isArray } from 'lodash';
-
+const GIT_POD_URL = `https://gitpod.io/#`;
 const GIT_REPO = 'https://github.com/estuary/flow-gitpod-base';
+const GIT_REPO_PATH = `/${GIT_REPO}`; // Must start with a slash to separate it from the variables
 
 // WARNING GitPod can change the URL format at anytime
-// https://www.gitpod.io/docs/configure/repositories/environment-variables#providing-one-time-environment-variables-via-the-context-url
-//  Last verified Q4 2024
+// https://www.gitpod.io/docs/classic/user/configure/workspaces/environment-variables#one-time-environment-variables-via-context-url
+//  Last verified Q3 2025
 export const generateGitPodURL = (
     draftId: string,
     token: RefreshTokenData,
-    derivationLanguage: string,
-    sourceCollections: Set<string> | string[],
     computedEntityName: string
 ) => {
-    const sourceCollectionCount = isArray(sourceCollections)
-        ? sourceCollections.length
-        : sourceCollections.size;
-
-    const gitPodURL = `https://gitpod.io/#`;
-
-    const urlVariables = `FLOW_DRAFT_ID=${encodeURIComponent(
-        draftId
-    )},FLOW_REFRESH_TOKEN=${encodeURIComponent(
-        Buffer.from(JSON.stringify(token)).toString('base64')
-    )},FLOW_TEMPLATE_TYPE=${derivationLanguage},FLOW_TEMPLATE_MODE=${
-        sourceCollectionCount > 1 ? 'multi' : 'single'
-    },FLOW_COLLECTION_NAME=${encodeURIComponent(computedEntityName)}`;
+    // https://github.com/estuary/flow-gitpod-base/blob/main/init.sh
+    // This object is consumed by GitPod init.sh and has to stay in sync
+    const settingsPart = `F_S=${encodeURIComponent(
+        Buffer.from(
+            JSON.stringify({
+                id: draftId,
+                rt: token,
+            })
+        ).toString('base64')
+    )}`;
```

Comment:
Keeping the settings that are actually used separate from the "uniqueProps" since those are not actually used.